pages/search.js

```diff
@@ -40,7 +40,9 @@ export default function Search() {
                                     name={repoOwner}
                                     title={repoName}
                                     projectType={projectType}
-                                    thumbnail={repoLogo}
+                                    thumbnail={
+                                        repoLogo ?? encodeURI(`https://og-image.vercel.app/${repoName}.png`)
+                                    }
                                     profileIcon={repoOwnerIcon}
```

Comment:
change this to ``profileIcon={`https://avatars.githubusercontent.com/${repoOwner}`}``